Dockerfile

```diff
@@ -18,15 +18,12 @@ COPY ./docker/nginx.conf ./docker/cors.conf /etc/nginx/
 
 # copy swagger files to the `/js` folder
 COPY ./dist/* /usr/share/nginx/html/
-COPY ./docker/run.sh /usr/share/nginx/
+COPY ./docker/docker-entrypoint.d/ /docker-entrypoint.d/
```

Comment:
It appears that `docker-entrypoint.d`  is only available with nginx 1.19 and later?

Reply:
Yes, you are right. This feature appeared on nginx 1.19.5. (https://github.com/nginxinc/docker-nginx/commit/56a20ba866321e5e5a3bd7dc2569d4a182e62506)

Do you think this is a problem? Because this Dockerfile builds on 1.21 …

Reply:
Hmm, well since the Dockerfile is indeed 1.21, guess this is a non-issue. 😄